### PR TITLE
Replace project state strings by round member variables

### PR DESCRIPTION
**Pull request by 70ray**

### CODE REVIEW

File: tools/proofers/for_mentors.php [hunk 1]
```diff
@@ -185,7 +185,7 @@ function output_project_details($mentored_round, $projectid, $nameofwork, $autho
 
 function get_beginner_projects_in_state($state, $mentored_round)
```
Comment (cpeel) [inline tools/proofers/for_mentors.php:186]: This is a good time to add types to this function def so phpstan can help us enforce this change.

File: tools/project_manager/page_compare.php [hunk 2]
```diff
@@ -167,10 +167,10 @@ public function radio_string($value, $label)
         return "<input type='radio' name='page_set' value='$value'$checked>" . html_safe($label);
     }
 
-    public function has_project_started_round($round_id)
+    private function has_project_started_round($round)
```
Comment (cpeel) [inline tools/project_manager/page_compare.php:170]: Ditto adding function type defs here to assist with this change.

Comment (70ray) [reply tools/project_manager/page_compare.php]: Done.

**Comment by 70ray** [comment]
Sandbox at: https://www.pgdp.org/~rp31/c.branch/use_defined_2

**Review by cpeel — APPROVED** [review]

**Review by srjfoo — APPROVED** [review]

**Pull request merged**